public/kibana-integrations/lib/format_hit.ts

```diff
@@ -36,10 +36,18 @@ const convert = (
   indexPattern: any,
   type: 'html' | 'text' = 'html'
 ): string => {
+  let format = { convert: () => val };
```

Comment:
**nitpick**: this seems not to be necessary, because the variable is set in the else statement and it is not used in the if statement.

Reply:
Fixed in https://github.com/wazuh/wazuh-kibana-app/pull/4925/commits/50aa541109011f72ab3ce52208981f9fc8be70e3.